client/src/components/employer-table-component/index.jsx

```diff
@@ -0,0 +1,318 @@
+import { useState } from "react";
+// import { useNavigate } from 'react-router-dom';
+import PropTypes from "prop-types";
+import {
+  TablePagination,
+  Box,
+  Button,
+  // IconButton,
+  Card,
+  CardContent,
+  TextField,
+  InputAdornment,
+  Typography,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Chip,
+  Avatar,
+} from "@mui/material";
+import AddIcon from "@mui/icons-material/Add";
+import SearchIcon from "@mui/icons-material/Search";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider, DatePicker } from "@mui/x-date-pickers";
+import { Container, DashboardContainer, HeaderContainer } from "./index.styles";
+
+function EmployerTableComponent({ employerData }) {
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+
+  const formatDate = (date) => {
+    return `${date.getMonth() + 1}/${date.getDate()}/${date.getFullYear()}`;
+  };
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  const slicedData = employerData.slice(
+    page * rowsPerPage,
+    page * rowsPerPage + rowsPerPage,
+  );
+
+  return (
+    <Container>
+      <DashboardContainer>
+        <HeaderContainer>
+          <ArrowBackIcon
+            sx={{
+              color: "gray",
+              marginRight: 2,
+              marginLeft: 2,
+              height: 35,
+              width: 35,
+              cursor: "pointer",
+            }}
+          />
+          <div style={{ flexGrow: 1 }}>
```

Comment:
I was wondering if you can split the three section into three different folders:

1. employer-dashboard-table
2. employer-dashboard-header
3. employer-dashboard-filter

similar to my pr here https://github.com/uoftblueprint/tces/pull/41

don't worry about logic, just have it render

something like this 
```
return (
   <MainContainer>
            <EmployerDashboardHeader/>
           <div>
                  <EmployerDashboardFilter/>
                 <EmployerDashboardTable/>
          </div>

</MainContainer>

)
```

rough sketch but something like that would be great